CFC_WebApp/clients/socialmedia/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "socialmedia",
+  "start_date": "2014/10/13",
+  "end_date": "2015/06/01",
+  "key": "dc23ad22-bc78-4fd6-9adf-ca2b04505916",
```

Comment:
You generally don't want the key to be checked in. It should be generated on each server.